docs/content/drafts/DropdownMenu2.mdx

```diff
@@ -0,0 +1,308 @@
+        <DropdownMenu.Overlay>
+          <ActionList>
+            <ActionList.Item selected={true}>Text</ActionList.Item>
+            <ActionList.Item>Number</ActionList.Item>
+            <ActionList.Item>Date</ActionList.Item>
+            <ActionList.Item>Iteration</ActionList.Item>
+          </ActionList>
+        </DropdownMenu.Overlay>
+      </DropdownMenu>
+    </>
+  )
+}
+
+render(<Example />)
+```
+
+### With Overlay Props
+
+```javascript live noinline
+// import {DropdownMenu, ActionList} from '@primer/react/drafts'
+const {DropdownMenu, ActionList} = drafts // ignore docs silliness; import like that ↑
+
+const fieldTypes = [
+  {icon: TypographyIcon, name: 'Text'},
+  {icon: NumberIcon, name: 'Number'},
+  {icon: CalendarIcon, name: 'Date'},
+  {icon: SingleSelectIcon, name: 'Single select'},
+  {icon: IterationsIcon, name: 'Iteration'}
+]
+
+const Example = () => {
+  const handleEscape = () => alert('you hit escape!')
+
+  const [selectedIndex, setSelectedIndex] = React.useState(1)
+  const selectedType = fieldTypes[selectedIndex]
+
+  return (
+    <DropdownMenu>
+      <DropdownMenu.Button aria-label="Select field type" leadingIcon={selectedType.icon}>
+        {selectedType.name}
+      </DropdownMenu.Button>
+      <DropdownMenu.Overlay width="medium" onEscape={handleEscape}>
+        <ActionList>
+          {fieldTypes.map((type, index) => (
+            <ActionList.Item key={index} selected={index === selectedIndex} onSelect={() => setSelectedIndex(index)}>
+              <type.icon /> {type.name}
+            </ActionList.Item>
+          ))}
+        </ActionList>
+      </DropdownMenu.Overlay>
+    </DropdownMenu>
+  )
+}
+
+render(<Example />)
+```
+
+### With a custom anchor
+
+You can choose to have a different _anchor_ for the Menu dependending on the application's context.
+
+&nbsp;
+
+```javascript live noinline
+// import {DropdownMenu, ActionList} from '@primer/react/drafts'
+const {DropdownMenu, ActionList} = drafts // ignore docs silliness; import like that ↑
+
+render(
+  <DropdownMenu>
+    <DropdownMenu.Anchor>
+      <button>Select a field type</button>
+    </DropdownMenu.Anchor>
+
+    <DropdownMenu.Overlay>
+      <ActionList>
+        <ActionList.Item selected={true}>Text</ActionList.Item>
+        <ActionList.Item>Number</ActionList.Item>
+        <ActionList.Item>Date</ActionList.Item>
+        <ActionList.Item>Iteration</ActionList.Item>
+      </ActionList>
+    </DropdownMenu.Overlay>
+  </DropdownMenu>
+)
+```
+
+<Note variant="warning">
+
+Use `DropdownMenu` to select an option from a small list. If you’re looking for filters or multiple selection, use [SelectPanel](/SelectPanel) instead.
+
+</Note>
+
+## Props / API reference
+
+### DropdownMenu
+
+| Name         | Type                           | Default | Description                                                                                                              |
```

Comment:
Why not use `PropsTable`?

Reply:
Yeah, would be super-helpful if we can switch this to the prop tables please 🙏

Reply:
Because this was created before PropsTable 😅 

Will update it now that we have it 🎉

Reply:
Done :)